accounts/views.py

```diff
@@ -1069,3 +1066,35 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
+    if settings.DEBUG:
+        paypal_validation_url = "https://www.sandbox.paypal.com/cgi-bin/webscr"
+    else:
+        paypal_validation_url = "https://www.paypal.com/cgi-bin/webscr"
+    req = requests.post(paypal_validation_url, data=params)
+    if req.text == 'VERIFIED':
+        campaign = DonationCampaign.objects.order_by('date_start').last()
```

Comment:
@ffont you can easily pass additional parameters to Paypal. CompaignID could be one of them. Then you can grab the campaign from paypal when you get the response...

I think assuming we only have one Campaign is kind of a bad programming idea, because... what if we ever want to run two campaigns at the same time? We might ask users to help us pay for <new design> and <some new feature> at the same time!